### PR TITLE
ENG-2298 Update blueprint to support bundle plugins with descriptorVersion 3

### PR DESCRIPTION
**Pull request by alepintus**

### CODE REVIEW

File: generators/server/templates/bundle/plugins/myplugin.yaml.ejs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+descriptorVersion: v3
```
Comment (avdev4j) [inline generators/server/templates/bundle/plugins/myplugin.yaml.ejs:1]: Do we need to do the same with the bundle descriptor and the widget descriptors?

Comment (alepintus) [reply generators/server/templates/bundle/plugins/myplugin.yaml.ejs:1]: @firegloves do I have to add the version also in the widget descriptors?

Comment (w-caffiero-entando) [reply generators/server/templates/bundle/plugins/myplugin.yaml.ejs:1]: @avdev4j @alepintus no, this update is limited to the plugin descriptor

Comment (firegloves) [reply generators/server/templates/bundle/plugins/myplugin.yaml.ejs:1]: no, only the plugin descriptor is interested by this change

Comment (avdev4j) [reply generators/server/templates/bundle/plugins/myplugin.yaml.ejs:1]: thanks :D

**Review by avdev4j — APPROVED** [review]

**Pull request merged**